react/week1/todo-app/src/todos.js

```diff
@@ -0,0 +1,53 @@
+const todoArray = [
+    {
+        id: 1,
+        task: "Write grocery list",
+        deadline: "Thu Feb 25 2021"
+    },
+    {
+        id: 2,
+        task: "Buy a gift for a firend",
+        deadline: "Fri Feb 26 2021"
+    },
+    {
+        id: 3,
+        task: "Book Dr appointment",
+        deadline: "Mon Mar 01 2021"
+    }
+]
+
+const Todoitems = ({ task, deadline }) => {
+    return (
+        <>
+            {task}, {deadline}
+        </>
+    )
+}
+
+const RenderTodos = ({ todos }) => {
+
+    return (
+        <ol>
+            {
+                todos.map(todo => {
+                    return (
+                        <li  key={todo.id}>
+                            <Todoitems task={todo.task} deadline={todo.deadline} />
```

Comment:
I don't know if this was covered in class, but lists in react should actually have a unique key. You can read more about those here: https://reactjs.org/docs/lists-and-keys.html#keys. If you would like to you can try to implement keys for your list.